src/features/marketplace/components/TradeableInfo.tsx

```diff
@@ -113,11 +143,25 @@ export const TradeableDescription: React.FC<{
             </Label>
           </div>
         )}
-        {tradeable && !tradeable?.isActive && (
+        {tradeable && !tradeable?.isActive && !tradeAt && (
           <div className="p-2 pl-0 pb-0">
+            <Label type="danger">{t("marketplace.notForSale")}</Label>
+          </div>
+        )}
+        {!canTrade && !!tradeAt && (
+          <div className="p-2 pl-0 pb-0 flex items-center justify-between">
             <Label type="danger" icon={SUNNYSIDE.icons.stopwatch}>
-              {t("marketplace.notForSale")}
+              {t("coming.soon")}
+            </Label>
+            <Label type="transparent">{formatDate(tradeAt)}</Label>
+          </div>
+        )}
+        {!canWithdraw && !!withdrawAt && (
+          <div className="p-2 pl-0 pb-0 flex items-center justify-between flex-wrap">
```

Comment:
<img width="215" alt="desktop" src="https://github.com/user-attachments/assets/ab306859-06f4-474c-9a82-1e7fc26d9bee" />

These may need to flex-col on desktop view - getting a bit squashed